src/app/components/Alert/styles.ts

```diff
@@ -0,0 +1,20 @@
+import { cva } from 'class-variance-authority';
+
+export const overlay = cva(
+  'bg-blackA9 data-[state=open]:animate-overlayShow fixed inset-0'
+);
+export const content = cva(
+  'data-[state=open]:animate-contentShow fixed top-[50%] left-[50%] max-h-[85vh] w-[90vw] max-w-[500px] translate-x-[-50%] translate-y-[-50%] rounded-[6px] bg-white p-[25px] shadow-[hsl(206_22%_7%_/_35%)_0px_10px_38px_-10px,_hsl(206_22%_7%_/_20%)_0px_10px_20px_-15px] focus:outline-none'
+);
```

Comment:
@belamacedo @karolinequetz uma coisa que vocês tem que ir pensando, é como vão fazer com os tokens que já estão fixos aqui e com a parte em "px".
Por exemplo, nesse alert, vocês colocaram bg-white nele, forçando que ele sempre será branco, será que faz sentido?
Se formos fazer aquele "modo noturno", isso já daria erro.